### DISCUSSION TITLE
Search context menu

### DISCUSSION THREAD

**Discussion by tvakisai** [microsoft-search/pnp-modern-search#2452] [Q&A]
Is it possible to replicate the functionality provided by the out of the box search context menu i.e. options to Open, Download, Open in Folder, Copy link etc (see attached image).

![PnPSearchOptions](https://user-images.githubusercontent.com/11034351/196202144-bd1e1011-4829-4700-91d7-a8db89d54b1a.PNG)

**Comment by FranckyC** [comment]
Hi, you could do it but using a custom web component. See https://microsoft-search.github.io/pnp-modern-search/extensibility/custom_web_component/
